### DISCUSSION TITLE
[Bug]: Issue with xterm

### DISCUSSION THREAD

**Discussion by vanemonni** [neurodesk/neurodesk.github.io-hugo#456] [Bug Report]
### Version

1.6.0

### What platform are you seeing the problem on?

Cloud

### What happened

Morning,

I have experiences issues with starting any software once logged in the remote frankfurt-neurodesk.org server (but using the desktop app). It used to work until a few days ago, now when I try to start any software I get this error message:

Failed to execute child process “xterm” (No such file or directory)

Also, I am connecting to a remote server because the local neurodesk option is not working anymore. So I am kind of stuck at the moment.

Any insights into how to solve this?

Thank you in advance!

**Comment by stebo85** [comment]
@iishiishii can you check if the new servers are set correctly in the app?

**Comment by stebo85** [comment]
@vanemonni yes, this could be a bug.

What error do you get when you start a local session? Are you running via docker or podman?

**Comment by stebo85** [comment]
@vanemonni - can you upgrade the Neurodeskapp on your side to 1.6.1? We updated the servers.

Also, can you describe why your local setup is not working? Then we can also help to get this back working :)
